core/templates/domain/topic/topic-update.service.spec.ts

```diff
@@ -581,27 +724,6 @@ describe('Topic update service', function() {
     expect(UndoRedoService.getCommittableChangeList()).toEqual([]);
   });
 
-  it('should set/unset changes to a topic\'s language code', function() {
```

Comment:
Why delete?

Reply:
Deleted because it was covered by the loop that dynamically generated tests I made earlier. Will revert

Reply:
Done